.cm/change_categories.cm

```diff
@@ -184,7 +184,7 @@ platforms:
       - 'subprojects/build-scan-performance/'
       - 'subprojects/enterprise/'
       - 'subprojects/enterprise-logging/'
-      - 'subprojects/enterprise-operations/'
+      - 'platforms/enterprise/enterprise-operations/'
```

Comment:
🤔 We keep this as `enterprise-operations` for now since it is only about the move, right? Would we move at some point to call the subdirectory only `operations`?

Reply:
I would do project renames once we move everything (enterprise) related to this platform-style structure